DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/DependencyInjection/AutofacRhinoModule.cs

```diff
@@ -63,7 +63,6 @@ protected override void Load(ContainerBuilder builder)
 
     // binding dependencies
     builder.RegisterType<CancellationManager>().InstancePerDependency();
-    builder.RegisterType<ServerTransport>().As<ITransport>().InstancePerDependency();
```

Comment:
this looks weird as part of this pr, but probably is related to something else...

Reply:
cools, merge away then!